### PR TITLE
XIVY-13137 Add cc:attrs for composites

### PR DESCRIPTION
**Pull request by ivy-lli**
Screenshot 2024-10-04 at 11 43 56

### CODE REVIEW

File: packages/protocol/src/data/form-data.ts [hunk 1]
```diff
@@ -68,3 +67,8 @@ export interface FormActionArgs {
   payload: string;
   context: FormContext;
 }
+
+export interface CompositeContext2 {
+  form: FormContext;
+  compositeId: string;
+}
```
Comment (ivy-lli) [inline packages/protocol/src/data/form-data.ts:74]: somehow this is not complete from the protocol generation

File: packages/protocol/src/data/form.ts [hunk 2]
```diff
@@ -35,6 +37,9 @@ export interface FormContext {
   file: string;
   pmv: string;
 }
+export interface CompositeContext {
+  compositeId: string;
+}
```
Comment (ivy-lli) [inline packages/protocol/src/data/form.ts:42]: form: FormContext missing

File: packages/protocol/src/data/form-data.ts [hunk 3]
```diff
@@ -68,3 +67,7 @@ export interface FormActionArgs {
   payload: string;
   context: FormContext;
 }
+
+export interface CompositeContext2 extends CompositeContext {
+  form: FormContext;
+}
```
Comment (ivy-rew) [inline packages/protocol/src/data/form-data.ts:73]: no longer needed :wink: 
https://jenkins.ivyteam.io/job/core_json-schema/job/XIVY-13137-ccParams/1/artifact/workspace/ch.ivyteam.ivy.dialog.form.lsp/target/schema/form/11.4.0/form-tsgen.json

**Review by ivy-rew — APPROVED** [review]

File: packages/protocol/src/data/index.ts
Comment (ivy-rew) [inline packages/protocol/src/data/index.ts:1]: thanks for the house-keeping

**Review by ivy-edp — COMMENTED** [review]
👍

**Pull request merged**